infrastructure/oss/inline_value.go

```diff
@@ -0,0 +1,40 @@
+/*
+ * © 2023 Snyk Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package oss
+
+import (
+	"github.com/rs/zerolog/log"
+
+	"github.com/snyk/snyk-ls/domain/snyk"
+)
+
+func (oss *Scanner) GetInlineValues(path string, myRange snyk.Range) ([]snyk.InlineValue, error) {
```

Comment:
nitpick: In this case not super important, but for structures it might be more efficient to use a pointers.